doc/static/manager.2b01f61c1c64d6482ce5.bundle.js

```diff
@@ -0,0 +1,32 @@
__(17),warning=__webpack_require__(2),didWarnForAddedNewProperty=!1,isProxySupported="function"==typeof Proxy,shouldBeReleasedProperties=["dispatchConfig","_targetInst","nativeEvent","isDefaultPrevented","isPropagationStopped","_dispatchListeners","_dispatchInstances"],EventInterface={type:null,target:null,currentTarget:emptyFunction.thatReturnsNull,eventPhase:null,bubbles:null,cancelable:null,timeStamp:function(event){return event.timeStamp||Date.now()},defaultPrevented:null,isTrusted:null};_assign(SyntheticEvent.prototype,{preventDefault:function(){this.defaultPrevented=!0;var event=this.nativeEvent;event&&(event.preventDefault?event.preventDefault():"unknown"!=typeof event.returnValue&&(event.returnValue=!1),this.isDefaultPrevented=emptyFunction.thatReturnsTrue)},stopPropagation:function(){var event=this.nativeEvent;event&&(event.stopPropagation?event.stopPropagation():"unknown"!=typeof event.cancelBubble&&(event.cancelBubble=!0),this.isPropagationStopped=emptyFunction.thatReturnsTrue)},persist:function(){this.isPersistent=emptyFunction.thatReturnsTrue},isPersistent:emptyFunction.thatReturnsFalse,destructor:function(){var Interface=this.constructor.Interface;for(var propName in Interface)"production"!=={NODE_ENV:"production",PUBLIC_URL:"."}.NODE_ENV?Object.defineProperty(this,propName,getPooledWarningPropertyDefinition(propName,Interface[propName])):this[propName]=null;for(var i=0;i<shouldBeReleasedProperties.length;i++)this[shouldBeReleasedProperties[i]]=null;"production"!=={NODE_ENV:"production",PUBLIC_URL:"."}.NODE_ENV&&(Object.defineProperty(this,"nativeEvent",getPooledWarningPropertyDefinition("nativeEvent",null)),Object.defineProperty(this,"preventDefault",getPooledWarningPropertyDefinition("preventDefault",emptyFunction)),Object.defineProperty(this,"stopPropagation",getPooledWarningPropertyDefinition("stopPropagation",emptyFunction)))}}),SyntheticEvent.Interface=EventInterface,"production"!=={NODE_ENV:"production",PUBLIC_URL:"."}.NODE_ENV&&isProxySupported&&(SyntheticEvent=new Proxy(SyntheticEvent,{construct:function(target,args){return this.apply(target,Object.create(target.prototype),args)},apply:function(constructor,that,args){return new Proxy(constructor.apply(that,args),{set:function(target,prop,value){return"isPersistent"===prop||target.constructor.Interface.hasOwnProperty(prop)||shouldBeReleasedProperties.indexOf(prop)!==-1||("production"!=={NODE_ENV:"production",PUBLIC_URL:"."}.NODE_ENV?warning(didWarnForAddedNewProperty||target.isPersistent(),"This synthetic event is reused for performance reasons. If you're seeing this, you're adding a new property in the synthetic event object. The property is never released. See https://fb.me/react-event-pooling for more information."):void 0,didWarnForAddedNewProperty=!0),target[prop]=value,!0}})}})),SyntheticEvent.augmentClass=function(Class,Interface){var Super=this,E=function(){};E.prototype=Super.prototype;var prototype=new E;_assign(prototype,Class.prototype),Class.prototype=prototype,
```

Comment:
__[eslint]__
* Error - Expected an assignment or function call and instead saw an expression. (no-unused-expressions)
* Warning - Missing function expression name. (func-names)
* Error - Wrap an immediate function invocation in parentheses. (wrap-iife)
* Error - Missing space before function parentheses. (space-before-function-paren)
* Error - Missing space before opening brace. (space-before-blocks)
* Error - Requires a space after '{'. (block-spacing)
* Error - Unexpected dangling '_' in '__webpack_require__'. (no-underscore-dangle)
* Error - Identifier '__webpack_require__' is not in camel case. (camelcase)
* Error - Expected space(s) after "if". (keyword-spacing)
* Error - 'installedModules' was used before it was defined. (no-use-before-define)
* Error - Expected space(s) before "return". (keyword-spacing)
* Error - Missing whitespace after semicolon. (semi-spacing)
* Error - All 'var' declarations must be at the top of the function scope. (vars-on-top)
* Error - Unexpected var
* Error - Infix operators must be spaced. (space-infix-ops)
* Error - A space is required after '{'. (object-curly-spacing)
* Error - Missing space before value for key 'exports'. (key-spacing)
* Error - Missing space before value for key 'id'. (key-spacing)
* Error - Missing space before value for key 'loaded'. (key-spacing)
* Error - A space is required before '}'. (object-curly-spacing)
* Error - Return statement should not contain assignment. (no-return-assign)
* Error - Unexpected use of comma operator. (no-sequences)
* Error - Missing semicolon. (semi)
* Error - Requires a space before '}'. (block-spacing)
* Error - Expected space(s) before "var". (keyword-spacing)
* Error - Strings must use singlequote. (quotes)
* Error - Using 'ForInStatement' is not allowed. (no-restricted-syntax)
* Error - Expected space(s) after "for". (keyword-spacing)
* Error - Expected space(s) before "if". (keyword-spacing)
* Error - Expected space(s) after "switch". (keyword-spacing)
* Error - Expected space(s) before "switch". (keyword-spacing)
* Error - Expected space(s) after "case". (keyword-spacing)
* Error - Assignment to property of function parameter 'modules'. (no-param-reassign)
* Error - Split 'var' declarations into multiple statements. (one-var)
* Error - Expected variable declaration to be on a new line. (one-var-declaration-per-line)
* Error - Unexpected comma in middle of array. (no-sparse-arrays)
* Error - Assignment to property of function parameter 'module'. (no-param-reassign)
* Error - '__webpack_require__' is defined but never used. (no-unused-vars)
* Error - Expected newline after "use strict" directive. (lines-around-directive)
* Error - 'use strict' is unnecessary inside of modules. (strict)
* Error - 'validateFormat' was used before it was defined. (no-use-before-define)
* Error - Expected 'undefined' and instead saw 'void'. (no-void)
* Error - Expected space(s) after "else". (keyword-spacing)
* Error - Unexpected function expression. (prefer-arrow-callback)
* Error - Unary operator '++' used. (no-plusplus)
* Error - Expected space(s) before "throw". (keyword-spacing)
* Error - 'format' is defined but never used. (no-unused-vars)
* Error - Expected literal to be on the right side of !==. (yoda)
* Error - Missing space before value for key 'NODE_ENV'. (key-spacing)
* Error - Missing space before value for key 'PUBLIC_URL'. (key-spacing)
* Error - Unexpected dangling '_' in '_len'. (no-underscore-dangle)
* Error - Unexpected dangling '_' in '_key'. (no-underscore-dangle)
* Error - Use the rest parameters instead of 'arguments'. (prefer-rest-params)
* Error - Unexpected string concatenation. (prefer-template)
* Error - 'args' used outside of binding context. (block-scoped-var)
* Error - Expected literal to be on the right side of !=. (yoda)
* Error - Expected '!==' and instead saw '!='. (eqeqeq)
* Warning - Unexpected console statement. (no-console)
* Error - Expected space(s) after "try". (keyword-spacing)
* Error - Expected space(s) after "catch". (keyword-spacing)
* Error - Expected space(s) before "catch". (keyword-spacing)
* Error - Empty block statement. (no-empty)
* Error - Unexpected dangling '_' in '_len2'. (no-underscore-dangle)
* Error - Unexpected dangling '_' in '_key2'. (no-underscore-dangle)
* Error - Use the spread operator instead of '.apply()'. (prefer-spread)
* Error - 'exports' is defined but never used. (no-unused-vars)
* Error - 'message' used outside of binding context. (block-scoped-var)
* Error - Expected literal to be on the right side of ===. (yoda)
* Error - Expected space(s) before "function". (keyword-spacing)
* Error - Expected space(s) after "return". (keyword-spacing)
* Error - Do not use String as a constructor. (no-new-wrappers)
* Error - Unexpected assignment within an 'if' statement. (no-cond-assign)
* Error - 'test2' used outside of binding context. (block-scoped-var)
* Error - 'source' is defined but never used. (no-unused-vars)
* Error - The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype. (guard-for-in)
* Error - 'to' used outside of binding context. (block-scoped-var)
* Error - Unexpected assignment within a 'for' statement. (no-cond-assign)
* Error - Unexpected dangling '_' in '_renderedComponent'. (no-underscore-dangle)
* Error - Assignment to function parameter 'component'. (no-param-reassign)
* Error - Unexpected dangling '_' in '_hostNode'. (no-underscore-dangle)
* Error - Assignment to property of function parameter 'node'. (no-param-reassign)
* Error - 'internalInstanceKey' was used before it was defined. (no-use-before-define)
* Error - Assignment to property of function parameter 'inst'. (no-param-reassign)
* Error - Unexpected dangling '_' in '_flags'. (no-underscore-dangle)
* Error - Unexpected use of '&'. (no-bitwise)
* Error - 'Flags' was used before it was defined. (no-use-before-define)
* Error - Unexpected dangling '_' in '_renderedChildren'. (no-underscore-dangle)
* Error - Unexpected labeled statement. (no-labels)
* Error - Using 'LabeledStatement' is not allowed. (no-restricted-syntax)
* Error - Do not access Object.prototype method 'hasOwnProperty' from target object. (no-prototype-builtins)
* Error - Unexpected dangling '_' in '_domID'. (no-underscore-dangle)
* Error - Unexpected mix of '&&' and '||'. (no-mixed-operators)
* Error - 'ATTR_NAME' was used before it was defined. (no-use-before-define)
* Error - Unexpected mix of '||' and '&&'. (no-mixed-operators)
* Error - Unexpected label in continue statement. (no-labels)
* Error - Unexpected use of continue statement. (no-continue)
* Error - 'invariant' was used before it was defined. (no-use-before-define)
* Error - '_prodInvariant' was used before it was defined. (no-use-before-define)
* Error - Unexpected use of '|='. (no-bitwise)
* Error - Assignment to function parameter 'node'. (no-param-reassign)
* Error - Expected space(s) before "for". (keyword-spacing)
* Error - 'parents' used outside of binding context. (block-scoped-var)
* Error - 'closest' used outside of binding context. (block-scoped-var)
* Error - Do not nest ternary expressions. (no-nested-ternary)
* Error - Unexpected dangling '_' in '_hostParent'. (no-underscore-dangle)
* Error - Assignment to function parameter 'inst'. (no-param-reassign)
* Error - Unexpected dangling '_' in '_prodInvariant'. (no-underscore-dangle)

<!-- Comment by [SideCI](https://sideci.com) -->